src/lib/components/search-attribute-filter/list-filter.svelte

```diff
@@ -11,6 +11,18 @@
 
   let list: string[] = [];
 
+  onMount(() => {
```

Comment:
How do you feel about this?

Reply:
Got [rid of the onMount](https://github.com/temporalio/ui/pull/2420/commits/b1bb5e0d512632e0295e07cf77927b707450d0bb) 🗑️